<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hospital</class>
 <widget class="QMainWindow" name="Hospital">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note: Please enter your original licence number, It cannot be changed.</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 213, 255, 255), stop:1 rgba(255, 0, 106, 255));
border-radius: 15px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="phone">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ropen">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>470</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mobile">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Licence no:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Registration open:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="licence">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>530</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Registration close:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Fax:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rclose">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>530</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Hospital open:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hopen">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>590</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fax">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>410</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>-10</y>
      <width>581</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 213, 255, 255), stop:1 rgba(255, 0, 106, 255));
border-radius: 10px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ventilators:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>570</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ambulance">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>570</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hclose">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>30</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Oxygen Cylinders:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ICU:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ambulance:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="oxygen">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Website</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Vaccination</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vaccination">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="website">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>450</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ventilator">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ICU">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="beds">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>510</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>MRI:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>No of beds</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>30</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Hospitla close:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MRI">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>390</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_patients">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>570</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Patients</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
